zebra/if_netlink.c

```diff
@@ -1143,15 +1143,15 @@ int netlink_link_change(struct sockaddr_nl *snl, struct nlmsghdr *h,
 			/* Update interface information. */
 			set_ifindex(ifp, ifi->ifi_index, zns);
 			ifp->flags = ifi->ifi_flags & 0x0000fffff;
-			if (IS_ZEBRA_IF_VRF(ifp))
```

Comment:
Again why this code move, there is no functional difference and no change in code paths.

Reply:
Same reasoning, zebra_if_set_ziftype() sets the zif_type (zebra interface type) which is used by the macro IS_ZEBRA_IF_VRF. Calling prior would not set VRF_LOOPBACK flag.